src/peergos/shared/user/fs/FileWrapper.java

```diff
@@ -109,6 +109,13 @@ public FileWrapper withVersion(Snapshot version) {
         return new FileWrapper(capTrie, pointer, linkPointer, entryWriter, ownername, version);
     }
 
+    @JsMethod
+    public CompletableFuture<FileWrapper> getLatest(NetworkAccess network) {
+        return network.synchronizer.getValue(owner(), writer())
+                .thenCompose(finished -> getUpdated(finished, network));
```

Comment:
maybe current or latest is a better name than finished?

Reply:
done